exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,17 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  return num < 0
+}
 
+function isBetween5and10(num) {
+  return num > 0
+}
+
+function startsWithD(name) {
+  return num < 0 && num > 0;
+}
```

Comment:
Did this code work Pamela? Or you are not finished yet?? I  copied and tried on my terminal just in case I am missing something, but it didn't work. I find that using "IF" statements help me a lot because i can see all my methods for solving the problem.... See how I did mine if you are interested to try...